src/main/java/org/sunbird/workflow/service/impl/NotificationServiceImpl.java

```diff
@@ -169,11 +174,67 @@ public void sendEmailNotification(WfRequest wfRequest) {
 		}
 	}
 
+	public void sendNotificationToMdoAdmin(WfRequest wfRequest) {
+		WfStatusEntity wfStatusEntity = wfStatusRepo.findByApplicationIdAndWfId(wfRequest.getApplicationId(),
+				wfRequest.getWfId());
+		WfStatus wfStatus = workflowservice.getWorkflowStates(wfStatusEntity.getRootOrg(), wfStatusEntity.getOrg(),
+				wfStatusEntity.getServiceName(), wfStatusEntity.getCurrentStatus());
+		try {
+			logger.info("Notification workflow status entity, {}", mapper.writeValueAsString(wfStatusEntity));
+			logger.info("Notification workflow status model, {}", mapper.writeValueAsString(wfStatus));
+		} catch (JsonProcessingException e) {
+			e.printStackTrace();
+		}
+		if (!ObjectUtils.isEmpty(wfStatus.getNotificationEnable()) && wfStatus.getNotificationEnable()) {
+			logger.info("Enter in the notification block");
+			List<Map<String, Object>> mdoAdminList = userProfileWfService.getMdoAdminDetails(wfRequest.getRootOrgId());
```

Comment:
You need only the list of email... why not return the same? what is the use to get userId and email for each mdo admin ??

Reply:
yes userId removed and also use list<String>